qiskit/providers/ibmq/utils/utils.py

```diff
@@ -56,3 +58,24 @@ def validate_job_tags(job_tags: Optional[List[str]], exception: Type[Exception])
     if job_tags and (not isinstance(job_tags, list) or
                      not all(isinstance(tag, str) for tag in job_tags)):
         raise exception("job_tags needs to be a list or strings.")
+
+
+def most_busy(backends: List['IBMQBackend']) -> 'IBMQBackend':  # type: ignore[name-defined]
```

Comment:
Per my other comment, this should be under the test directory, and I'd prefer a more explicit name like `most_busy_backend`, with `provider` being the argument.

Reply:
Ah! That'd be more consistent with its use case, being that we only use it in test. Went ahead and moved that.

Initially, I passed in the list of backends, since that would give us more flexibility with the specific backend we want to get (has more than `n` qubits, is/not is a simulator, etc). But, now that I think of it, we won't really need to specific backends for our use case. I'll make those changes as well!